Aggiunta funzione per il calcolo delle prestazioni in autorotazione

giulioguarino opened the pull request.

Commit:
Creazione funzione per il calcolo delle prestazioni in autorotazione

Commit:
creazione funzione

Commit:
First attempt

Commit:
piccola correzione versione precedente

Commit:
commento migliorato

Commit:
Aggiunta test case

giulioguarino commented:
We have two doubts about the function.

1) Solving  the first equation (in respect to theta_0) yields two solutions. Is there a way to choose the one with physical meaning? 
   We arbitrarily chose the second one because it seemed more reasonable.

 2) The code doesn't seem to work for values of advance ratio which are equal to or less than 0.1. We suspect this might be because of the theoretical limits of the procedure described in reference.

EttoreSaetta commented on prestazioni_autorotazione.m:
```diff
@@ -0,0 +1,149 @@
+function [T_C, H_C, alpha_deg, omega, V_inf ] = prestazioni_autorotazione(mu,X)
+
+%Supposing helicopter data:
+global  R...                          % [m], rotorblade radius
```
Do not use global variables.

EttoreSaetta commented on prestazioni_autorotazione.m:
```diff
@@ -0,0 +1,149 @@
+% |Note        : note addizionali                                                              |
+% ==============================================================================================
+
+function [T_C, H_C, alpha_deg, omega, V_inf ] = prestazioni_autorotazione(mu,X)
```
The name of the function should be in English.

EttoreSaetta reviewed (commented):
Some comments here, you also need to upload a documentation for starting a review.
Moreover put the test case in an appropriate folder.
As regards your theoretical questions we can discuss about it during our meetings.
You can join our Tuesday meetings for these questions.

Commit:
funzione rinominata; 
eliminazione variabili global con inserimento diretto dei dati del rotore negli input della funzione.

Commit:
Inserimento cartella TestCase.

Commit:
Correzione testcase.

Commit:
Aggiunta cartella TestCases

Commit:
piccole correzioni, aggiunta di un warning

Commit:
aggiunta di un secondo test case

Commit:
creazione documentazione

giulioguarino commented:
This should be the definitive version of the function. We are waiting for a final review.

Commit:
correzione documentazione.
